docs/sources/enterprise/access-control/manage-rbac-roles.md

```diff
@@ -277,46 +246,102 @@ curl --location --request POST '<grafana_url>/api/access-control/roles/' \
+   | `name`                | The name of the basic role you want to update. You can specify a `uid` instead of a role name. The role `name` or the `uid` are required. |
+   | `orgId`               | Identifies the organization to which the role belongs. `global` can be used instead to specify it's a global role.                        |
+   | `version`             | Identifies version of the role, to prevent overwriting newer changes.                                                                     |
+   | `from`                | List of roles to copy permissions from.                                                                                                   |
+   | `permissions > state` | The state of the permission. You can set it to `absent` to ensure its exclusion from the copy list.                                       |
 
 1. Reload the provisioning configuration file.
 
    For more information about reloading the provisioning configuration at runtime, refer to [Reload provisioning configurations]({{< relref "../../http_api/admin/#reload-provisioning-configurations" >}}).
 
-The following example removes the `fixed:permissions:admin` from the `Grafana Admin` basic role.
+The following example modifies the `Grafana Admin` basic role permissions.
+It removes from it, the permission to list, grant and revoke roles to teams.
+It also adds to it the permission to read and write grafana folders.
 
 ```yaml
 # config file version
-apiVersion: 1
+apiVersion: 2
 
-# list of default basic role assignments that should be removed
-removeDefaultAssignments:
-  - builtInRole: 'Grafana Admin'
-    fixedRole: 'fixed:permissions:admin'
+roles:
+  - name: 'basic:grafana_admin'
+    global: true
+    version: 3
+    from:
+      - name: 'basic:grafana_admin'
+        global: true
+    permissions:
+      # Permissions to remove
+      - action: 'teams.roles:list'
+        scope: 'teams:*'
+        state: 'absent'
+      - action: 'teams.roles:remove'
+        scope: 'permissions:type:delegate'
+        state: 'absent'
+      - action: 'teams.roles:add'
+        scope: 'permissions:type:delegate'
+        state: 'absent'
+      # Permissions to add
+      - action: 'folders:read'
+        scope: 'folder:*'
+      - action: 'folders:write'
+        scope: 'folder:*'
 ```
 
-You can also remove fixed roles from basic roles using the API. Refer to the [RBAC HTTP API]({{< relref "../../http_api/access_control.md#remove-a-built-in-role-assignment" >}}) for more details.
+> **Note**: You can add multiple `fixed`, `basic` or `custom` roles to the `from` section. Their permissions will be copied and added to the basic role.
+> <br/> > **Note**: Make sure to **increment** the role version for the changes to be accounted for.
+
+You can also change basic roles' permissions using the API. Refer to the [RBAC HTTP API]({{< relref "../../http_api/access_control.md#update-a-role" >}}) for more details.
+
+## Reset basic roles permissions to factory
```

Comment:
I've seen 'factory' elsewhere and don't think we should use that word. Our software was not designed in a factory. Can we simply say 'default settings' or something like that?